### DISCUSSION TITLE
Add option to remove "Begin backup" button from file explorer

### DISCUSSION THREAD

**Discussion by Mik3l24** [valinet/ExplorerPatcher#2776] [Ideas]
Microsoft added a really distracting OneDrive promotional message to the explorer that has been bugging me for weeks: (screenshot is in Polish, but I hope you get what I mean)
![obraz](https://github.com/valinet/ExplorerPatcher/assets/52327131/3901d4ce-451b-47cd-b664-fcf8ccb5bedd)
It takes up a ridiculous amount of space in the address bar and at some window sizes it even completely obscures it. I guess it does get reduced to just an icon when clicked on, but it resets on opening a new instance.

It appears only on folders that are part of a "Library" (Documents, Pictures etc.) and I presume OneDrive needs to be installed and running. I do use it (just not for backup), so I don't want to uninstall the whole OneDrive to get rid of it.

So, would it be possible to add an option to this program to remove this button completely?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Amrsatrio** [comment]
You can use the Windows 7 command bar or Windows 10 ribbon interface to get rid of that button. Moreover it's more responsive.

**Reply by Mik3l24** [reply]
I do like the look of the Windows 11 one... (controversial opinion, I know) though I understand this could serve as a temporary stopgap till this annoyance is properly removed from the W11 version.

So my feature request is essentially to add a way to remove the button shown in the original post from the original Windows 11 address bar.

I'll get around to posting the version and build numbers a bit later, I can't do this at this moment (and I forgot about it earlier)

By the way, I'm wondering how this application works. I presume it does binary hacking of explorer's machine code or something like that?

**Reply by pyrates999** [reply]
The source code is available.  So you can find out yourself.
